src/client/content_cao.cpp

```diff
@@ -630,14 +644,16 @@ void GenericCAO::addToScene(ITextureSource *tsrc)
 		m_meshnode->setMaterialFlag(video::EMF_BILINEAR_FILTER, false);
 		m_meshnode->setMaterialType(material_type);
 		m_meshnode->setMaterialFlag(video::EMF_FOG_ENABLE, true);
-	}
-	else if(m_prop.visual == "mesh") {
+	} else if(m_prop.visual == "mesh") {
 		infostream<<"GenericCAO::addToScene(): mesh"<<std::endl;
 		scene::IAnimatedMesh *mesh = m_client->getMesh(m_prop.mesh, true);
 		if(mesh)
 		{
```

Comment:
can you fix that conditional code style as you modified the 3 lines after ?

Reply:
Still seems uncorrected.
@pauloue

Reply:
@paramat @pauloue Fix pushed. I thought nerzhul was talking about the space after the 'if', sorry. My personal style is Allman-like, so I easily miss these things because they look natural to me.